dependencies.ini

```diff
@@ -0,0 +1,63 @@
+# Instructions:
+#
+# add library: library = pinned.version.number
+# add with no pinned version: library <blank>
+# remove a library: library = remove
+#
+# end instructions
+
+##### Typical installation (conda)
+# main libraries from main conda
+[core]
+h5py = 2.9.0
+numpy = 1.16.4
+scipy = 1.2.1
+scikit-learn = 0.21.2
+pandas = 0.24.2
+xarray = 0.12.1
+netcdf4 = 1.4.2
+matplotlib = 3.0.3
+statsmodels = 0.9.0
+cloudpickle = 1.1.1
+python
+hdf5
+swig
+pylint
+coverage
+lxml
+psutil
+pip
+importlib_metadata
+
+# secondary installs from conda forge
+[forge]
+pyside2
+
+# tertiary installs directly from pip
+[pip]
+lazy-import
+
+# optional libraries
+[optional]
+tensorflow = 1.13.1
+pillow = 6.0.0
+
+# operating system alterations
+[windows]
+# nothing currently
+
+[mac]
+# nothing currently
+
+[linux]
+# these prevent Intel crash errors with mkl
+nomkl
+numexpr
+
+##### Alternate installs
+# install using pip
```

Comment:
Could you add more descriptions about how we formulate the list of libraries for different situations? For example, when use Conda to install: we need 'core', 'forge', and 'pip'. For pip-install, we need 'core', 'forge', and 'pip', but remove 'pip-install'

Reply:
I tried to capture that with the `##### Typical Installation` and `##### Alternate installs` structure, then `core, secondary, tertiary` and `optional, windows, mac, linux`. Let me know how you think this could be clarified.

I feel any additional logic should be contained in the `library_handler.py`; otherwise, it has to be maintained in two places. Let me know if more comments should be added to clarify that.

Reply:
I went ahead and added some comments anyway; let me know if that's along the lines you were thinking.